Paymetheus/ViewModels/ImportScriptDialogViewModel.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) 2016 The btcsuite developers
+// Copyright (c) 2016 The Decred developers
+// Licensed under the ISC license.  See LICENSE file in the project root for full license information.
+
+using Grpc.Core;
+using Paymetheus.Decred.Util;
+using Paymetheus.Framework;
+using System;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Input;
+
+namespace Paymetheus.ViewModels
+{
+    public sealed class ImportScriptDialogViewModel : DialogViewModelBase
+    {
+        private DelegateCommandAsync _importScript;
+        public ICommand Execute => _importScript;
+
+        public ImportScriptDialogViewModel(ShellViewModel shell) : base(shell)
+        {
+            _importScript = new DelegateCommandAsync(ImportScriptAction);
+            _importScript.Executable = false;
+        }
+
+        private byte[] _scriptBytes;
+        public string ScriptHexString {
+            get { return Hexadecimal.Encode(_scriptBytes); }
+            set {
+                var scriptValid = false;
+                try
+                {
+                    _scriptBytes = Hexadecimal.Decode(value);
+                    scriptValid = true;
+                }
+                finally
+                {
+                    _importScript.Executable = scriptValid;
+                }
+            }        
+        }
+        public string Passphrase { private get; set; } = "";
+
+        private async Task ImportScriptAction()
+        {
+            try
+            {
+                await App.Current.Synchronizer.WalletRpcClient.ImportScriptAsync(_scriptBytes, true, 0, Passphrase);
```

Comment:
It's hard to say. The importscript dialogue box should have them, but the only reason this was bothered to be added was so that people could get stats on their pool tickets. I'll just disable rescan for now. The hook to the API should definitely have access to the ability to set rescan, and what height to set it to.